Support CC0 license by adding CC0 template

SabiRini opened the pull request.

uzzu commented on plugin/src/main/resources/template/licenses/cc0.html:
```diff
@@ -0,0 +1,147 @@
+  <% print library.url.isEmpty() ? "" : """<p><a href="${library.url}">${library.url}</a></p>""" %>
+  <div class="license">
+    <h2>CC0 1.0 Universal</h2>
+    <p>${library.copyrightStatement}</p>
```
Do not needed because CC0 is no copyright.
https://creativecommons.org/publicdomain/zero/1.0/

uzzu commented:
ref: #38

SabiRini commented:
Hi @uzzu :) I've updated my pull request.
I've added the cc0 example to the build.gradle and yml file, as well as generated an updated JSON and HTML. Hope everything is ok. Thanks for feedback.

Best, SabiRini

uzzu commented:
Perfect. Thank you :)
